### ISSUE TITLE
question about checkbox 

### ISSUE THREAD

**Issue by bung87** [adrsch/slider-captcha#27]
the checkbox with "I am human" is a simple checkbox without human behavior detection , what's the purpose here ?

btw I've forked this repo rewrites to typescript , including mono repo  toolkit etc... https://github.com/ad2302/slider-captcha
if you interested take it or accept PR for this , please tell me.

**Comment by bravo-kernel** [comment]
Good question, could we show the slider after an event. E.g. after user presses a `Post` button?

**Comment by bung87** [comment]
in my real use case , the captcha show up after user fill phone number, after captcha pass server will send sms code to user

**Comment by bravo-kernel** [comment]
Exactly, similar to what I want to do. I like your typescript version too btw 💯 but... cannot not open issues on your repository.

Also, a React example using `create` and `verify` functions (instead of URL) would be very helpful.

**Comment by bung87** [comment]
> Exactly, similar to what I want to do. I like your typescript version too btw 💯 but... cannot not open issues on your repository.
> 
> Also, a React example using `create` and `verify` functions (instead of URL) would be very helpful.

now I enabled issue feature.
